string target_model_name
string origin_model_name 
string origin_link_name
---
bool ok

